### PR TITLE
DG fix

### PR DESCRIPTION
**Pull request by adamwth**

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -1693,46 +1693,69 @@ Expected: Checks-in the guest Pikachu's booking and adds him to the checked-in g
...
-** `reassign r/001 from/32/11/18 nr/002` (no such date)
-** `reassign r/005 from/12/11/18 nr/101` (no such rooms)
+** `reassign r/001 from/32/11/18 nr/002` (invalid command, no such date)
+** `reassign r/005 from/12/11/18 nr/101` (invalid command, no such new room)
+** `reassign r/006 from/1/1/18 nr/007` (room 006 has no such booking)
+** `reassign r/006 from/9/11/18 nr/007` (cannot reassign booking, because room 006 booking is expired)
+
+* Refer to the user guide to test more edge cases
+
+[NOTE]
```
Comment (pikulet) [inline docs/DeveloperGuide.adoc]: Haha I think there only needs to be one note of this but it's good.

**Review by pikulet — APPROVED** [review]
👍
